src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.CI.yaml

```diff
@@ -2,6 +2,41 @@
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHjnbqe9AmEW1Js10nySybyuAG7Fb5E9EHUgkmqFDv7PxQGajxbkewesj3PCq3IiGvEkAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMLdJMjR0tj3iBnE3zAgEQgDuyIU42lpmTEO6PIHEU1dyNeIx0UiVTdoUqn4nXWcCOsqlRNWM1o7CyByuiATE23j7d9AIOhwEZxCRvew==
 config:
+  heroku:app_id:
+    secure: v1:d1JeftBc68QcCrCH:tQV5P2EU4H/eF7jMjUied4fIcj2Q6d9ujnhvfn5oxJiPbh41oVff6sWQDs9SUl65gQOTWg==
+  heroku:user: "mitx-devops"
+  heroku_app:vars:
+    AWS_STORAGE_BUCKET_NAME: "ol-xpro-app-ci"
+    CYBERSOURCE_REFERENCE_PREFIX: "xpro-ci"
```

Comment:
These should be able to just be templated in the Python code since it's just a difference of the env stage.